### DISCUSSION TITLE
OWSM shared model and tokenizer

### DISCUSSION THREAD

**Discussion by Papazola43** [espnet/espnet#5487] [Q&A] [answered]
Hello everyone,
I'm here to ask about running OWSM V1 and V2 on my device.
Apparently, I found that OWSM V1's model parameter isn't shared in huggingface and even thou OWSM V2 has the shared model parameter, OWSM V2's tokenizer is missing the token.txt file.
Could you please share the pretrained weight for the OWSM V1 along with token.txt files for OWSM V1 and OWSM V2?
Thank you very much

**Comment by sw005320** [comment, marked as answer]
@pyf98, can you upload them?

@Papazola43, note that OWSM V1 is an initial version with limited training data (but it still has 40K hours), and the performance would be limited, but I think it is still good to upload. Thanks for your request.

**Comment by pyf98** [comment]
Hi @Papazola43 I will upload v1 soon.
But I do not understand the question about token.txt. What is that? Do you need it?

**Reply by pyf98** [reply]
I think we do not upload a separate token list. It actually exists in the config file.

**Reply by Papazola43** [reply]
Hi @pyf98 ,
I think there is some misunderstanding on my side, so I apologize.
Maybe I will post another discussion regarding fine tuning OWSM which is related to this problem. Thank you very much!

**Reply by alumae** [reply]
I think I know what the original author meant: we need `data/token_list/bpe_unigram50000/tokens.txt` in order to finetune the models. I think we can really extract it from config file but it's a bit wkward.

**Comment by pyf98** [comment]
The v1 model is now at: https://huggingface.co/espnet/owsm_v1

**Reply by Papazola43** [reply]
Thanks @pyf98 !
